### PR TITLE
/me please for the love of God KILL ME!!!

### PR DESCRIPTION
**Pull request by KingofHONK**
Inspired by every shitter mime who has ever /me'd for a bag of holding, the faguette is a horrible abomination cursed by the Gods. 

image

Creates the mime equivalent of a cluwne, aptly named a faguette (for it has a baguette for an arm) Enjoy.
[content] [honk]
Thank to fellow dumbass for help with sprite 

:cl:
 * rscadd: Added sprites for faguettes to animal.dmi
 * rscadd: Added faguettes to /mob/living/simple_animal/hostile/retaliate/faguette

### CODE REVIEW

**Comment by ShiftyRail** [comment]
You didn't update the .dme

**Comment by Kitfox88** [comment]
is that sans undertale

also can you cut off the baguette arm and eat it

**Comment by KingofHONK** [comment]
>also can you cut off the baguette arm and eat it

gunna seperate PR that soon(tm)

**Comment by Gringonius** [comment]
Shouldn't the name be more in like with 'cluwne'?

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	/mob/living/simple_animal/hostile/retaliate/faguette/to_bump(atom/movable/AM as mob|obj)
+		if(now_pushing)
+			return
+		if(ismob(AM))
+			var/mob/M = AM
+			to_chat(src, "<span class='danger'>You are too depressed to push [M] out of \the way.</span>")
+			M.LAssailant = src
+			return
+	..()
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm:59]: Un-indent every line after 51 except this one by one

**Comment by Dacendeth** [comment]
When you butcher one of these you should get a baguette instead of meat.

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+/mob/living/simple_animal/hostile/retaliate/faguette/to_bump(atom/movable/AM as mob|obj)
+	if(now_pushing)
+		return
+	if(ismob(AM))
+		var/mob/M = AM
+		to_chat(src, "<span class='danger'>You are too depressed to push [M] out of \the way.</span>")
+		M.LAssailant = src
+		return
+..()
```
Comment (Exxion) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm]:
```suggestion
	..()
```

File: code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+/mob/living/simple_animal/hostile/retaliate/faguette/proc/handle_disabilities()
+	if ((prob(5) && paralysis < 10))
+		to_chat(src, "<span class='warning'>You have a seizure!</span>")
+		Paralyse(10)
+
+/mob/living/simple_animal/hostile/retaliate/faguette/emote(var/act, var/type, var/message, var/auto)
+	if(timestopped)
+		return //under effects of time magick
+
+	var/msg = pick("drools through its stitched mouth","silently cries into [gender==MALE?"his":"her"] baguette","sloppily mimes tieing an invisible noose around its neck")
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/hostile/retaliate/faguette.dm]: \his should be an inbuilt macro that does this more directly

**Comment by Gringonius** [comment]
What about the name though

**Comment by Intigracy** [comment]
faguette is perfect

**Comment by Kurfursten** [comment]
Just don't tell Github because they will freak

**Comment by SonixApache** [comment]
image

**Comment by SonixApache** [comment]
YOU FORGOT TO CHECK THE FILE IN THE MAIN .DME

**Pull request merged**